vms/event/tests/test_event.py

```diff
@@ -0,0 +1,154 @@
+# third party
+from selenium import webdriver
+
+# Django
+from django.contrib.staticfiles.testing import LiveServerTestCase
+
+# local Django
+from pom.pages.authenticationPage import AuthenticationPage
+from pom.pages.eventsPage import EventsPage
+from shift.utils import (create_admin, create_event_with_details)
+
+
+class EventDetails(LiveServerTestCase):
+    """
+    Contains Tests for Job app in aspect of
+    an admin's view of website.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        cls.driver = webdriver.Firefox()
+        cls.driver.implicitly_wait(5)
+        cls.driver.maximize_window()
+        cls.event_details_page = EventsPage(cls.driver)
+        cls.authentication_page = AuthenticationPage(cls.driver)
+        super(EventDetails, cls).setUpClass()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.driver.quit()
+        super(EventDetails, cls).tearDownClass()
+
+    def setUp(self):
+        create_admin()
+        self.login_admin()
+
+    def tearDown(self):
+        pass
+
+    @staticmethod
+    def register_valid_event():
+        created_event = create_event_with_details(['event', '2050-06-11', '2050-06-19'])
+        return created_event
+
+    def check_error_messages(self):
+        event_details_page = self.event_details_page
+        error_message = 'This field is required.'
```

Comment:
Make this a variable inside `EventsPage` itself (multiple uses might be possible.)

Reply:
Done.